ui/camera.py

```diff
@@ -561,6 +561,27 @@ def draw(self, context):
         layout.prop(contour_lines, "steps")
         layout.prop(contour_lines, "zero_grid_size")
 
+class LUXCORE_CAMERA_PT_image_pipeline_white_balance(CameraButtonsPanel, Panel):
+    bl_label = "White Balance"
+    bl_parent_id = "LUXCORE_CAMERA_PT_image_pipeline"
+    bl_options = {'DEFAULT_CLOSED'}
+    lux_predecessor = "LUXCORE_CAMERA_PT_image_pipeline_contour_lines"
```

Comment:
If we change the order during export, it should be changed here as well (should have `"LUXCORE_CAMERA_PT_image_pipeline_vignetting"` as predecessor).

Looking at these panels, they seem to be a bit jumbled around, I can clean that up after the pull request is merged.